app/models/like.rb

```diff
@@ -0,0 +1,12 @@
+class Like < ApplicationRecord
+  belongs_to :user
```

Comment:
I'd suggest that your like model should also `belongs_to :author, class_name: 'User'` not just user